File: server/RegistrarSuite.DTO/Metadata/CountryDto.cs
using RegistrarSuite.DTO.Common;

namespace RegistrarSuite.DTO.Metadata
{
    public class CountryDto
    {
        public string? ShortCode { get; set; }
        public string? ShortName { get; set; }
        public string? NativeName { get; set; }
        public string? Flag { get; set; }
        public string? CurrencyCode { get; set; }
        public string? CallingCode { get; set; }
        public string? Latitude { get; set; }
        public string? Longitude { get; set; }
        public decimal Population { get; set; }
    }

    public class CountryDrpDto : Dropdown
    {
        public string? ShortCode { get; set; }
        public string? NativeName { get; set; }
        public string? Flag { get; set; }
    }

}

File: client/registrar-suite/src/@core/dto/CountryDto.ts
export class CountryDto {
  shortCode?: string;
  shortName?: string;
  nativeName?: string;
  flag?: string;
  currencyCode?: string;
  callingCode?: string;
  latitude?: string;
  longitude?: string;
  population: number | undefined;
}

export class CountryDrpDto {
  shortCode?: string;
  nativeName?: string;
  flag?: string;
}

